apps/ledger-live-mobile/src/screens/ClaimNft/ClaimNftQrScan.tsx

```diff
@@ -163,9 +165,7 @@ const ClaimNftQrScan = () => {
   return (
     <SafeAreaView style={{ flex: 1 }} edges={["bottom"]}>
       <Flex flex={1}>
-        {!permission?.canAskAgain &&
-        !permission?.granted &&
-        permission?.status === "denied" ? (
```

Comment:
This is unrelated to the fix, just a small cleanup:
`permission.granted` is a helper that abstracts into a boolean `permission.status`, in this specific case checking the value of `permission.granted` is enough.